source/fuzz/fuzzer_pass_add_parameters.h

```diff
@@ -23,8 +23,8 @@ namespace spvtools {
 namespace fuzz {
 
 // Randomly decides for each non-entry-point function in the module whether to
-// add new parameters to it. If so, randomly determines the number of parameters
-// to add, their type and creates constants used to initialize them.
+// add one new parameter to it. If so, randomly determines parameter's type and
+// creates a constant to initialize it.
```

Comment:
The comment needs to be fixed to make it clear that many parameters are added - can revert to something similar to the old comment.

Reply:
Fixed in 93b8e3a.